### ISSUE TITLE
master_authorized_networks_config.value is object with 1 attribute "cidr_blocks"

### ISSUE THREAD

**Issue by marinkovicpetar** [terraform-google-modules/terraform-google-composer#42]
### TL;DR

I am trying to provision v2 environment with `master_authorized_networks` variable, but it's not working for me. I double checked the repository, and I think that I am defininig this variable in a proper way as described here https://github.com/terraform-google-modules/terraform-google-composer/blob/master/modules/create_environment_v2/variables.tf#L220=

### Expected behavior

Provision the environment with `master_authorized_networks` configuration working

### Observed behavior

```
│ Error: Invalid index
│
│   on .terraform\modules\composer\modules\create_environment_v2\main.tf line 141, in resource "google_composer_environment" "composer_env":
│  141:             cidr_block   = master_authorized_networks_config.value["cidr_block"]
│     ├────────────────
│     │ master_authorized_networks_config.value is object with 1 attribute "cidr_blocks"
│
│ The given key does not identify an element in this collection value.
╵
╷
│ Error: Invalid index
│
│   on .terraform\modules\composer\modules\create_environment_v2\main.tf line 142, in resource "google_composer_environment" "composer_env":
│  142:             display_name = master_authorized_networks_config.value["display_name"]
│     ├────────────────
│     │ master_authorized_networks_config.value is object with 1 attribute "cidr_blocks"
│
│ The given key does not identify an element in this collection value.
```

### Terraform Configuration

```hcl
module "composer" {
  source                           = "terraform-google-modules/composer/google//modules/create_environment_v2"
  version                          = "3.0.0"
  region                           = "us-east1"
  project_id                       = "my-gcp-project"
  composer_env_name                = "composer-sandbox"
  network                          = "my-gcp-network"
  subnetwork                       = "my-gcp-subnetwork"
  environment_size                 = "ENVIRONMENT_SIZE_SMALL"

  scheduler = {
    cpu        = 0.5
    memory_gb  = 1.875
    storage_gb = 1
    count      = 1
  }

  web_server = {
    cpu        = 0.5
    memory_gb  = 1.875
    storage_gb = 1
  }

  worker = {
    cpu        = 0.5
    memory_gb  = 1.875
    storage_gb = 1
    min_count  = 1
    max_count  = 3
  }

  master_authorized_networks = [
    {
        cidr_block   = "5.5.5.5/32",
        display_name = "Bastion host"
    }
  ]
}
```
```


### Terraform Version

```sh
Terraform v1.1.7
on windows_amd64
+ provider registry.terraform.io/hashicorp/google v4.9.0
+ provider registry.terraform.io/hashicorp/google-beta v4.9.0

Your version of Terraform is out of date! The latest version
is 1.1.8. You can update by downloading from https://www.terraform.io/downloads.html
```
```


### Additional information

_No response_

**Comment by cheddarwhizzy** [comment]
#47

**Comment by bharathkkb** [comment]
fixed by https://github.com/terraform-google-modules/terraform-google-composer/pull/47
